platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -55,11 +55,21 @@ public static synchronized Mapbox getInstance(@NonNull Context context, @NonNull
    * @return Mapbox Access Token
    */
   public static String getAccessToken() {
+    validateMapbox();
     validateAccessToken();
     return INSTANCE.accessToken;
   }
 
   /**
+   * Runtime validation of Mapbox creation.
+   */
+  private static void validateMapbox() throws InvalidAccessTokenException {
+    if (INSTANCE == null) {
+      throw new InvalidAccessTokenException();
```

Comment:
@tobrun Could we have a different message thrown by this `InvalidAccessTokenException` so that the developer can differentiate between and error from `validateAccessToken` and an error `validateAccessToken`?